VogtPayroll/Employee.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtPayroll
+{
+    class Employee
+    {
+        private string _name;
+        private int _totalWorkingHours;
+        private decimal _payrate;
+
+        public Employee(string name, int totalWorkingHours, decimal payRate)
+        {
+            _name = name;
+            _totalWorkingHours = totalWorkingHours;
+            _payrate = payRate;
+        }
+
+        public Employee GetEmpInfo()
+        {
+            // allow user to enter employee name, total working hours and payrate
+
+            Console.WriteLine("What is the employee's name?");
+            var empName = Console.ReadLine();
+
+            Console.WriteLine("What is the employee's total working hours?");
+            var empTotalWorkingHours = Convert.ToInt32(Console.ReadLine());
+
+            Console.WriteLine("What is the employee's payrate?");
+            var empPayRate = Convert.ToDecimal(Console.ReadLine());
+
+            Employee employee = new Employee(empName, empTotalWorkingHours, empPayRate);
+
+            return employee;
+
+        }
+
+        public void ComputeGrossPay(Employee employee)
```

Comment:
1. Make all of these compute functions private, since (from what I can tell from looking at the Chegg problem) you'll only need to use them within the PrintEmpInfo function -- which will already have access to these functions.
2. Make all of these compute functions RETURN the computation. So change the return statement to decimal.